flow/layers/container_layer.cc

```diff
@@ -26,14 +26,22 @@ void ContainerLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
 void ContainerLayer::PrerollChildren(PrerollContext* context,
                                      const SkMatrix& child_matrix,
                                      SkRect* child_paint_bounds) {
+  const bool had_platform_view = context->has_platform_view;
```

Comment:
nit: I'd keep the simpler version of this where we assume platform view has no children (it is a leaf layer)

Reply:
Done. I did put in a debug assert just to try and catch in case this assumption ever changes in the future.